### PR TITLE
fix: #653 update notifications and modals

### PR DESCRIPTION
**Pull request by J0taFerreira**
Updated dialog and toast to use primevue

### CODE REVIEW

**Review by NickSaglioni — CHANGES_REQUESTED** [review]
Thanks for the work Jota. Just a couple of comments.

File: frontend/src/App.vue
[the remark this answers is not in the document]
Comment (J0taFerreira) [reply frontend/src/App.vue:1]: they are now removed, thanks!

File: frontend/src/assets/styles/default-theme.scss
[the remark this answers is not in the document]
Comment (J0taFerreira) [reply frontend/src/assets/styles/default-theme.scss:1]: just updated to use the design tokens

File: frontend/src/components/Dashboard.vue [hunk 1]
```diff
@@ -302,6 +308,17 @@ async function tryDelete(assignment: FamApplicationUserRoleAssignmentGet) {
...
+// .access-table {
+//     margin-top: 100px;
+//     background: #ffffff;
+//     border-radius: 4px 4px 0px 0px;
+//     border: 2px solid #dfdfe1;
+// }
+// .table-header {
+//     padding: 16px 16px 24px;
+// >>>>>>> 5dd49b0 (fix/653-update-notifications-and-modals: Added dialog and success message to dashboard)
+// }
```
[the remark this answers is not in the document]
Comment (J0taFerreira) [reply frontend/src/components/Dashboard.vue]: did, thanks!

**Review by NickSaglioni — APPROVED** [review]

**Pull request merged**